scripts/gitlab/check_runtime.sh

```diff
@@ -127,34 +123,46 @@ then
 
 fi
 
-
-# Introduce runtime/polkadot/src/lib.rs once Polkadot mainnet is live.
-for VERSIONS_FILE in runtime/kusama/src/lib.rs
+# Iterate over each runtime defined at the start of the script
+for RUNTIME in "${runtimes[@]}"
 do
+
+	# Check if there were changes to this specific runtime (or common).
+	# If not, we can skip to the next runtime
+	if ! git diff --name-only "refs/tags/${LATEST_TAG}...${CI_COMMIT_SHA}" \
+		| grep -E -q -e "^runtime/common|^runtime/${RUNTIME}"; then
+		continue
+	fi
+
 	# check for spec_version updates: if the spec versions changed, then there is
 	# consensus-critical logic that has changed. the runtime wasm blobs must be
 	# rebuilt.
 
-	add_spec_version="$(git diff refs/tags/${LATEST_TAG}...${CI_COMMIT_SHA} ${VERSIONS_FILE} \
-		| sed -n -r "s/^\+[[:space:]]+spec_version: +([0-9]+),$/\1/p")"
-	sub_spec_version="$(git diff refs/tags/${LATEST_TAG}...${CI_COMMIT_SHA} ${VERSIONS_FILE} \
-		| sed -n -r "s/^\-[[:space:]]+spec_version: +([0-9]+),$/\1/p")"
+	add_spec_version="$(
+		git diff "refs/tags/${LATEST_TAG}...${CI_COMMIT_SHA}" "runtime/${RUNTIME}/src/lib.rs" \
+		| sed -n -r "s/^\+[[:space:]]+spec_version: +([0-9]+),$/\1/p"
+	)"
+	sub_spec_version="$(
+		git diff "refs/tags/${LATEST_TAG}...${CI_COMMIT_SHA}" "runtime/${RUNTIME}/src/lib.rs" \
+		| sed -n -r "s/^\-[[:space:]]+spec_version: +([0-9]+),$/\1/p"
+	)"
 
 
 	# see if the version and the binary blob changed
 	if [ "${add_spec_version}" != "${sub_spec_version}" ]
 	then
 
-		if git diff --name-only origin/master...${CI_COMMIT_SHA} \
-			| grep -q -e '^runtime/'
+		if git diff --name-only "origin/master...${CI_COMMIT_SHA}" \
+			| grep -q -e "^runtime/$RUNTIME" -e '^runtime/common'
 		then
 			# add label breaksapi only if this pr altered the runtime sources
-		  github_label "D2-breaksapi"
+			github_label "D2-breaksapi"
```

Comment:
didn't see that before, but for this function to work the corresponding labels have to be whitelisted. this hasn't been done when the label changed and labelling hasen't happend by this ever since then. see: https://gitlab.parity.io/parity/infrastructure/github-api/-/blob/master/.gitlab-ci.yml#L27

will update that repo now.